containers-ordered-strict/src/Map/Ord.hs

```diff
@@ -4,26 +4,26 @@ module Map.Ord
        ( Map
        , Key
 
-       , empty
-       , singleton
-       , fromList
-
-       , null
-       , size
-       , member
-       , lookup
-       , lookupDefault
```

Comment:
This function is called `lookupDefault` in the signature, so, unfortunately, this particular function you can't export as `M.findWithDefault`. It should be written in previous explicit way.